### ISSUE TITLE
Option to customise or remove animation on Navigator push or pop.

### ISSUE THREAD

**Issue by SethuSenthil** [tomalabaster/flutter_app_lock#20]
**Is your feature request related to a problem? Please describe.**
I request biometrics to unlock my app through flutter_app_lock. Since this is sensitive information, I also make sure to hide the contents of the app when the app is backgrounded (with the same UI displaying the app's logo which I implemented for flutter_app_lock). When the user launches the app, the contents of the screen are already hidden, then the app lock pushes to a new page that looks identical to the previous page, however it has a sliding animation. When the user successfully authenticates themself, it pops back to the previous page which is identical. 

TLDR; when the app has a splash screen to hide the app content when app is backgrounded when using the same splash screen with app lock (say for quick biometric), it causes the app to go back and forth to which appears to be "the same page" to the end-user.
 
This causes unnecessary complications in the UI the end-user does not need to see. 

**Describe the solution you'd like**
An option to disable the Navigator sliding animations.

**Describe alternatives you've considered**
I have seen https://stackoverflow.com/questions/49874272/how-to-navigate-to-other-page-without-animation-flutter
however, since this package uses pushReplacementNamed and not pushReplacement, I'm not sure how to implement the "Duration = 0 " solution.

**Additional context**

**Comment by tomalabaster** [comment]
Hi! Thanks for raising. I'm looking to remove the use of a `Navigator` or `MaterialApp` completely in issue #8 and with that offer the ability to use your own transitions, this should be suitable to your needs!

**Comment by tomalabaster** [comment]
Acknowledging that this is still an issue that v4.0.0 doesn't resolve, but looking to resolve :+1:

**Comment by tomalabaster** [comment]
@SethuSenthil actually does >=4.1.0 resolve this by being able to use the `inactiveBuilder` instead of implementing your own?

**Comment by tomalabaster** [comment]
@SethuSenthil have you had a chance to review version >=4.1.0?
